Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationOption.m

```diff
@@ -77,6 +78,23 @@ - (instancetype)initWithName:(NSString *)name iconName:(BPKIconName)iconName tag
     return self;
 }
 
+- (instancetype)initWithName:(NSString *)name
```

Comment:
Could `initWithName:(NSString *)name iconName:(BPKIconName)iconName tag:(NSInteger)tag` just call this method now?